src/gatsby/src/components/advisories/advisoryFilter.js

```diff
@@ -101,6 +157,17 @@ const AdvisoryFilter = ({
                   }
                 }}
               />
+              {filterText.length > 0 &&
+                <div className="rbt-aux">
+                  <button 
+                    area-label="Clear" 
+                    className="close btn-close rbt-close" 
+                    onClick={handleClearKeyword}
+                  >
+                    ×
```

Comment:
Used the same x from `react-bootstrap-typeahead`. Just wanted to make it look consistent with other search fields' x button since the `react-bootstrap-typeahead` button is not really customizable 🤷‍♀️